packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -316,14 +317,44 @@ class DocumentCollection {
   /**
    * Use Couch _changes API
    *
-   * @param  {String} since     Starting sequence for changes
-   * @param  {Object} options   { includeDesign: false, includeDeleted: false }
+   * @param  {Object} couchOptions Couch options for changes https://kutt.it/5r7MNQ
+   * @param  {Object} options      { includeDesign: false, includeDeleted: false }
    */
-  async fetchChanges(since, options = {}) {
-    const result = await this.stackClient.fetchJSON(
-      'GET',
-      `/data/${this.doctype}/_changes?include_docs=true&since=${since}`
-    )
+  async fetchChanges(couchOptions = {}, options = {}) {
```

Comment:
I am not a fan of having to options parameters.

COuld there be only one ?

```
async fetchChanges(options) {

}
```

Reply:
I prefer don't add breaking change on this PR.

Reply:
I can be implemented in a non breaking way, like the way the parameter is checked to be a string.

```
async fetchChanges(options, options2) {
  // Support for old ways of calling fetchChanges
  const opts = {}
  if (typeof options === 'object') {
   Object.assign(opts, options)  
  }
  if (typeof options === 'string') {
    opts.filter = options
  }
  if (typeof options2 === 'object') {
     Object.assign(opts, options2)
   }
   

   // Rest of the method, relying on a normalized opts
  // object  
   

}

```